sys/net/network_layer/sixlowpan/ip.c

```diff
@@ -890,10 +890,11 @@ uint16_t ipv6_csum(ipv6_hdr_t *ipv6_header, uint8_t *buf, uint16_t len,
                    uint8_t proto)
 {
     uint16_t sum = 0;
+    static char addr_str2[IPV6_MAX_ADDR_STR_LEN];
     DEBUG("Calculate checksum over src: %s, dst: %s, len: %04X, buf: %p, proto: %u\n",
```

Comment:
What about using the following to prevent the introduction of another variable?
```c
DEBUG("Calculate checksum over src: %s, ",
    ipv6_addr_to_str(addr_str, IPV6_MAX_ADDR_STR_LEN, &ipv6_header->srcaddr));
DEBUG("dst: %s, len: %04X, buf: %p, proto: %u\n",
    ipv6_addr_to_str(addr_str, IPV6_MAX_ADDR_STR_LEN, &ipv6_header->destaddr),
len, buf, proto);
```